### ISSUE TITLE
FB10960832: when an app requires Accessibility and the user allows, they are redirected to the Appearance Settings instead of Security & Privacy

### ISSUE THREAD

**Issue by godbout** [feedback-assistant/reports#345]
- **Date:** 2022-07-31
- **Resolution:** Open
- **Area:** AppKit
- **OS:** Ventura public beta 2
- **Type:** Incorrect/Unexpected Behavior

## Description

my apps need Accessibility. i use the commonly used API to do this: https://developer.apple.com/documentation/applicationservices/1459186-axisprocesstrustedwithoptions#

the dialog shows up. i press `Allow`. i'm being sent to the Settings, Appearance, while i should be sent to Security & Privacy -> Accessibility.

thanks.

## Files

https://user-images.githubusercontent.com/121373/182020880-6ff8c102-6a25-4692-8c37-43afcacf97a1.mp4
